src/postamble_minimal.js

```diff
@@ -124,10 +124,10 @@ var wasmModule;
 if (!WebAssembly.instantiateStreaming && !Module['wasm']) throw 'Must load WebAssembly Module in to variable Module.wasm before adding compiled output .js script to the DOM';
 #endif
 (WebAssembly.instantiateStreaming
-  ? WebAssembly.instantiateStreaming(fetch('{{{ TARGET_BASENAME }}}.wasm'), imports)
+  ? WebAssembly.instantiateStreaming(fetch(Module['wasm'] || '{{{ TARGET_BASENAME }}}.wasm'), imports)
```

Comment:
I believe `Module['wasm']` is supposed to represent the actual wasm module object. 

I'm sure @juj will have some idea about how to acheive what you want here.. without negatively impacting code size.

Reply:
Yeah, I wanted something along the lines of locateFile for minimal runtime, but that doesn't seem to exist. Module.wasm indeed refers to a typed array/buffer rather than a url, should it then be `Module.locateFile() || ` instead?

Reply:
Yes, something like that I guess.. but @juj will have a better idea about how to achieve that in `MINIMAL_RUNTIME`

Reply:
yup, that'd be great, I'm hacking this quite a bit, as I'm running minimal runtime across 3 different WASM files, from the same worker, which is why I needed the conditional URL, with MINIMAL_RUNTIME_WASM_URL I could pretty much do the same thing and set it as Module["wasm"]